ostd/src/bus/mmio/common_device.rs

```diff
@@ -55,12 +55,12 @@ impl MmioCommonDevice {
 
     /// Device ID
     pub fn device_id(&self) -> u32 {
-        self.io_mem.read_val::<u32>(8).unwrap()
+        self.io_mem.read_once::<u32>(8).unwrap()
     }
 
     /// Version of the MMIO device.
     pub fn version(&self) -> VirtioMmioVersion {
-        VirtioMmioVersion::try_from(self.io_mem.read_val::<u32>(4).unwrap()).unwrap()
+        VirtioMmioVersion::try_from(self.io_mem.read_once::<u32>(4).unwrap()).unwrap()
```

Comment:
Again, let's call this method `read_version` to differentiate it from a simple getter method.

Reply:
Same as above. See ["Rename some misleading method names"](https://github.com/asterinas/asterinas/pull/1231/commits/92ec012baf9c91a10246ebc58304efb3796dcaec).